Src/BootCamp.Chapter/LinqMethodDemo.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class LinqMethodDemo
```

Comment:
This is not a demo- it's an actual extension. I would call it IEnumerableExtensions

Reply:
done